lib/Controller/DocumentController.php

```diff
@@ -159,6 +159,9 @@ private function setupPolicy($response) {
 		$policy = new ContentSecurityPolicy();
 		$policy->addAllowedFrameDomain($wopiDomain);
 		$policy->allowInlineScript(true);
+		if (method_exists($policy, 'addAllowedFormActionDomain')) {
```

Comment:
`addAllowedFormActionDomain` exists since Nextcloud 17 so the check is not needed.

Reply:
Should I remove it from `lib/AppInfo/Application.php` too, then?

Reply:
Yes, you may.

Reply:
Done.